### ISSUE TITLE
Doesn't work at all

### ISSUE THREAD

**Issue by romu70** [Laverna/laverna#847]
Since yesterday, reaching https://laverna.cc/app/#/notes doens't work.

I run Fedora 26. I tried with Firefox 55 and 57 Developer Edition.

On FF 55 : green screen "Loading..."
On FF 57 : All the interface is there, but there is no notes nor notebooks.

**Comment by Frankstar** [comment]
working fine for me.

**Comment by romu70** [comment]
Do you sync with DropBox @Frankstar? Just tried in private mode to setup from scratch.

Same the sync symbol turns for few seconds, and then stop, Laverna content stays blank.

**Comment by slaponicus** [comment]
Same here. I get the following error in the JS console:

`Error: Script error for "dropbox", needed by: modules/dropbox/classes/sync
http://requirejs.org/docs/errors.html#scripterror
require.js:5:1060`

Using FF Developer edition and dropbox sync. Works fine in Chrome however.

**Comment by rSSi** [comment]
Same here. FF 56, 64Bit, Elementary OS
Also, works fine in Chromium

**Comment by wwebfor** [comment]
Try to clear cache. I think it's because of cache.

**Comment by romu70** [comment]
Done, no change.

**Comment by romu70** [comment]
Just installed Chromium, no change, same problem.

**Comment by wwebfor** [comment]
I can't reproduce it. I have tried in Firefox Developer edition 57 and Firefox 56.0.1.

**Comment by romu70** [comment]
I think I got it.

Historically, my Laverna storage in DropBox was in Applications\RTLaverna. Now, it's in Applications\Laverna. So, I copied/pasted the content of the notes-db folder from the old RTLaverna to Laverna, and now it works.

**Comment by romu70** [comment]
Same on my 2nd PC. Flushed the cache and offline data. Laverna starts, shows the DropBox popup and then redirects to : https://www.dropbox.com/oauth2/authorize?response_type=token&client_id=m2dhs7ohtkzzh7e&redirect_uri=https://laverna.cc/app/#notes

Error 400.

**Comment by slaponicus** [comment]
After clearing site data in FF developer edition, my console error is gone and I can access the app now. 

@romu70, from the look of the dropbox error it looks like something wasn't configured correctly when the app got set up for your dropbox account? I am redirected to a similar page with the same redirect_uri and did not receive a 400 error.

**Comment by romu70** [comment]
Yes, there is something I can't explain. I deleted the application in Dropbox. I deleted the "RTLaverna" which stood in the Applications folder in my Dropbox. I cleared the Firefox cache and local data.

But if I get to the Laverna page, I still see my notes. There is something I miss obviously, but what?

**Comment by wwebfor** [comment]
@romu70 , if I understood correctly, you are using your own Dropbox API key. If so, check step N6 in [this wiki](https://github.com/Laverna/laverna/wiki/FAQ#how-to-connect-laverna-with-dropbox)

**Comment by romu70** [comment]
Hi, that's what I did yesterday evening, sorry I didn't post any reply. I totally deleted the application within DropBox and recreated it from scratch and re-ran the Laverna setup, and now it works.
